Mathlib/CategoryTheory/Monoidal/Types/Coyoneda.lean

```diff
@@ -0,0 +1,61 @@
+/-
+Copyright (c) 2018 Michael Jendrusch. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Jendrusch, Scott Morrison
+
+! This file was ported from Lean 3 source module category_theory.monoidal.types.coyoneda
+! leanprover-community/mathlib commit 95a87616d63b3cb49d3fe678d416fbe9c4217bf4
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.CategoryTheory.Monoidal.Types.Basic
+import Mathlib.CategoryTheory.Monoidal.CoherenceLemmas
+
+/-!
+# `(𝟙_ C ⟶ -)` is a lax monoidal functor to `Type`
+-/
+
+
+open CategoryTheory
+
+open CategoryTheory.Limits
+
+open Tactic
+
+universe v u
+
+namespace CategoryTheory
+
+open Opposite
+
+open MonoidalCategory
+
+-- Porting note: made it noncomputable.
+-- `failed to compile definition, consider marking it as 'noncomputable' because it`
+-- `depends on 'CategoryTheory.typesMonoidal', and it does not have executable code`
+-- I don't know if that is a problem, might need to change it back in the future, but
+-- if so it might be better to fix then instead of at the moment of porting.
+
+/-- `(𝟙_ C ⟶ -)` is a lax monoidal functor to `Type`. -/
+noncomputable
```

Comment:
Is this a problem to simply add `noncomputable`? This has one dependent file. If it is a problem there, it might be easier to fix it when porting that leaf instead of now.

Reply:
Not at problem!